blaze/expr/arithmetic.py

```diff
@@ -50,6 +50,55 @@ def name(o):
         return None
 
 
+class Op(ElemWise):
+    def __init__(self, ops):
+        if not isinstance(ops, (list, tuple,)):
```

Comment:
This should be an ABC or something that has 2 subclasses, `UnaryOp` and `BinaryOp`. This means that we can isolate the responsibilities of each class